core/src/main/java/hudson/model/AbstractItem.java

```diff
@@ -255,6 +255,7 @@ public boolean isNameEditable() {
     @RequirePOST
     @Restricted(NoExternalUse.class)
     public HttpResponse doConfirmRename(@QueryParameter String newName) throws IOException {
+
```

Comment:
Please revert unrelated whitespace or formatting changes, to keep the diff as short as possible. Assists review.